tests/resources/presence_response/presence_payload.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
it becomes more difficult to me to see the exact payload returned by a ws

Reply:
For me it's a choice. I thought that centring it like this avoided adding json strings in tests making them bulkier. If you wish to do the same as java, tell me and I will.

I plan to check if adding scope=module for the fixture makes use of the same json for all tests. (we wouldn't open the file for each test)

Reply:
@symphony-elias what's your take on this? (putting all the mocked responses in the same file)